drivers/mpathcount

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+
+# Copyright (C) Citrix Systems Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation; version 2.1 only.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+import os
+import sys
+import re
+import json
+import subprocess
+
+from sm.core import util
+from sm.core import xs_errors
+from sm.core import mpath_cli
+from sm import mpathcount
+
+if __name__ == '__main__':
+    try:
+        session = util.get_localAPI_session()
+    except:
+        print("Unable to open local XAPI session")
+        sys.exit(-1)
+
+    localhost = session.xenapi.host.get_by_uuid(mpathcount.get_localhost_uuid())
+    mpathcount.check_xapi_is_enabled()
+    # Check whether multipathing is enabled (either for root dev or SRs)
+    try:
+        if mpathcount.get_root_dev_major() != mpathcount.get_dm_major():
+            hconf = session.xenapi.host.get_other_config(localhost)
+            assert(hconf['multipathing'] == 'true')
+            mpath_enabled = True
+    except:
+        mpath_enabled = False
```

Comment:
This should be initialised to False  above the try as the `if` might evaluate False

Reply:
No, this one should be addressed as mpathcount.mpath_enabled...